core/src/main/scala/io/finch/Finch.scala

```diff
@@ -0,0 +1,16 @@
+package io.finch
+
+import cats.effect.Effect
+import io.finch.endpoint.effect.EffectEndpoints
+import io.finch.syntax.effect.EffectEndpointMappers
+
+abstract class Finch[F[_]](implicit val E: Effect[F])extends EffectEndpoints[F]
```

Comment:
I think we should name this `Module` by analogy with iteratee.

Reply:
`Module` is 👍